src/core/core.layouts.js

```diff
@@ -397,35 +371,23 @@ module.exports = {
 			fullWidth: chartArea.w,
 			chartArea: chartArea,
 			vBoxMaxWidth: (width - chartAreaWidth) / verticalBoxes.length,
+			maxPadding: maxPadding,
 			hBoxMaxHeight: chartHeight / 2 // (height - chartArea.h) / horizontalBoxes.length
 		};
 
-		// Step 4
-		// Sum of box minimum dimensions per side {top,left,bottom,right}
-		var outerDims = calculateMinSizes(outerBoxes, params);
-
-		// Find out greatest padding required per side {top,left,bottom,right}
-		params.maxPadding = findMaxPadding(outerBoxes);
-
-		// Calculate parameters (chart area dimensions needed in next step)
-		updateWithMaxPadding(params, outerDims);
+		// Step 4 - skipped
```

Comment:
Can we remove this and re-number the steps instead?

Reply:
The descriptions for steps 5-6 should probably be updated as well as they no longer make sense

Reply:
removed the numbering and changed comments